quesma/testdata/clients/kunkka.go

```diff
@@ -347,8 +346,8 @@ var KunkkaTests = []testdata.AggregationTestCase{
 									"doc_count": 0
 								},
 								"doc_count": 2,
-								"key": 1718794800000,
-								"key_as_string": "2024-06-19T11:00:00.000"
+								"key": 1718787600000,
+								"key_as_string": "2024-06-19T09:00:00.000"
```

Comment:
Here and in a ton of other tests: if it's 11am bucket like here, in Warsaw time, then we returned 11:00, but it displayed bar at 13:00, because Kibana always interprets times in UTC (11 UTC = 13 Warsaw). Now 9:00 UTC = 11:00 Warsaw and it's good.